src/pages/public_registration/LogPanel.tsx

```diff
@@ -44,22 +46,39 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     staleTime: Infinity,
     gcTime: 1_800_000, // 30 minutes
   });
-
-  const userQuery = useQuery({
-    enabled: !!resourceOwnerId,
-    queryKey: ['user', resourceOwnerId],
-    queryFn: resourceOwnerId ? () => fetchUser(resourceOwnerId) : undefined,
-    retry: 0,
-    staleTime: Infinity,
-    gcTime: 1_800_000, // 30 minutes
-  });
+  const organizationAcronym = organizationQuery.isPending
+    ? ''
+    : organizationQuery.data
+      ? organizationQuery.data?.acronym
+      : t('common.unknown');
 
   const logs: LogItem[] = [];
 
+  if (isImported) {
+    registration.importDetails?.map((importDetail) => {
```

Comment:
Siden du ikke vil bruke outputten her (annet en `push` som du gjør manuelt), så tror jeg du vil ha `forEach` her
```suggestion
    registration.importDetails?.forEach((importDetail) => {
```